### PR TITLE
Dashboard sprint 4

### PR DESCRIPTION
**Pull request by jelbin313**
Added dashboard page, changed icons for views, made buttons and margins uniform on monthly, weekly, and tasks page.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -24,9 +24,17 @@
...
-        return render_template('tasks.html')
+        return render_template('dashboard.html')
     return render_template("homepage.html")
 
+@app.get('/dashboard')
+def dashboard():
+    if session.get('user_authenticated'):
+        return render_template("dashboard.html", user=session['user_email'])
+    flash('You need to be logged in first', category='error')
+    return render_template("login.html")
```
Comment (sagarsubedi) [inline app.py:35]: I think it was brought up in a different issue but I would argue that redirect seems better than rendering.

File: templates/tasks.html [hunk 2]
```diff
@@ -160,19 +171,19 @@ <h1>Upcoming</h1>
...
-        '          type="text" autofocus placeholder="Add an item..."/>'+
-        '      </span>' + 
-        '  </td>' +
-        '  <td style="width:72px">' +
-        '    <span id="filler-'+x.id+'" class="material-icons">more_horiz</span>' + 
-        '    <span id="save_edit-'+x.id+'" hidden class="save_edit material-icons">done</span>' + 
-        '    <span id="undo_edit-'+x.id+'" hidden class="undo_edit material-icons">cancel</span>' +
-        '  </td>' +
-        '</tr>';
+    // t = '<tr id="task-'+x.id+'" class="task">' +
```
Comment (sagarsubedi) [inline templates/tasks.html:174]: Any reason why you pushed a commented-out code? Just wondering.

File: templates/dashboard.html [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+            </div>
+        </div>
+
+        <!--div for the views section-->
+        <div class="row dash_margin task_border">
+            <div class="task_col dash_padding">
+                <h1>Views</h1>
+                <p>
+                    There are 3 views that can be used to see your calendar. The first is the monthly view, which is
```
Comment (sagarsubedi) [inline templates/dashboard.html:33]: Would you agree that this is too much text for a dashboard? I think of a dashboard as a page for quick data or summary or things like that. Something of an "at a glance" data display. That is why I think this is too much text for a dashboard.

**Review by sagarsubedi — APPROVED** [review]

**Review by SeanDonnelly1 — APPROVED** [review]

File: templates/tasks.html (hunk 2 above)
Comment (Qtipshooter) [reply templates/tasks.html:174]: This can be deleted soon I believe

**Pull request merged**